lib/Runtime/Library/JavascriptBigInt.h

```diff
@@ -4,44 +4,46 @@
 //-------------------------------------------------------------------------------------------------------
 #pragma once
 
+
 namespace Js
 {
     class JavascriptBigInt sealed : public RecyclableObject
     {
-
     private:
-        Field(uint32*) m_digits;         // digits
-        Field(uint32) m_length;          // length
+        Field(digit_t*) m_digits;         // digits
+        Field(digit_t) m_length;          // length
+        Field(digit_t) m_maxLength;          // max length without resize
         Field(bool) m_isNegative;
 
         // TODO: BigInt should be arbitrary-precision, need to resize space and add tagged BigInt
```

Comment:
could remove this TODO since you're doing it :)

Reply:
will remove